storage/ctab/src/main/java/org/openscience/cdk/io/RdfileReader.java

```diff
@@ -131,15 +131,6 @@ public RdfileReader(InputStream in, IChemObjectBuilder chemObjectBuilder, boolea
         this(new InputStreamReader(in, StandardCharsets.UTF_8), chemObjectBuilder, continueOnError);
     }
 
-    /**
-     * Creates a new RdfileReader instance with the given InputStream and IChemObjectBuilder.
-     *
-     * @param reader the Reader providing the RDfile data
-     */
-    public RdfileReader(Reader reader) {
-        this(reader, IChemObjectBuilder.find(), true);
-    }
-
```

Comment:
@johnmay This does change a public API. Totally acceptable imo if a `2.10.1` release is put out in the next few weeks as this was only introduced in `2.10.`0, but otherwise just removing it might break someone's code...

Adding a public `RdfileReader(Reader,IChemObjectBuilder)` seems reasonable to me so users don't have to think about parameters such as `continueOnError` and can go with a default.

Up to you, just wanted to voice my opinion :)

Reply:
Broken is broken -  AFAIK it wouldn't work without a builder right?

Reply:
Good point, I agree.